src/index.js

```diff
@@ -0,0 +1,132 @@
+import {} from 'https://cdn.jsdelivr.net/npm/@woowacourse/mission-utils@1.0.1/dist/mission-utils.min.js';
+
+export default class BaseballGame {
+  constructor() {
+    this.randomNumbers = this.getRandomNumbers();
+    this.submitBtn = document.getElementById('submit');
+    this.result = document.getElementById('result');
+    this.input = document.getElementById('user-input');
+    this.init();
+    console.log(this.randomNumbers);
+  }
+
+  init() {
+    this.result.innerText = '';
+    this.handleSubmit();
+  }
+
+  getRandomNumbers() {
+    const [MIN, MAX, LEN] = [1, 9, 3];
+    return MissionUtils.Random.pickUniqueNumbersInRange(MIN, MAX, LEN);
+  }
+
+  ElemOfArrToString(arr) {
+    return arr.map(el => el.toString());
+  }
+
+  play(computerInputNumbers, userInputNumbers) {
+    const comInput = this.ElemOfArrToString(computerInputNumbers);
+    const userInput = this.ElemOfArrToString(userInputNumbers);
+    const ballNumbers = this.getBallNumbers(comInput, userInput);
+    const strikeNumbers = this.getStrikeNumbers(comInput, userInput);
+    const ballStr = ballNumbers === 0 ? '' : `${ballNumbers}볼 `;
+    const strikeStr = strikeNumbers === 0 ? '' : `${strikeNumbers}스트라이크`;
+
+    if (strikeNumbers === 3) {
+      return '정답';
+    }
+    if (strikeNumbers === 0 && ballNumbers === 0) {
+      return '낫싱';
+    }
+    return (ballStr + strikeStr).trim();
+  }
+
+  getBallNumbers(answer, input) {
+    let ballNumbers = 0;
+    answer.map((el, i) => {
+      if (el !== input[i] && input.includes(el)) {
+        ballNumbers += 1;
+      }
+    });
+    return ballNumbers;
+  }
+
+  getStrikeNumbers(answer, input) {
+    let strikeNumbers = 0;
+    answer.map((el, i) => {
+      if (el === input[i]) {
+        strikeNumbers += 1;
+      }
+    });
+    return strikeNumbers;
+  }
+
+  isCorrectInput(input) {
+    const isDuplicate = input.some(
+      x => input.indexOf(x) !== input.lastIndexOf(x),
+    );
+    const isThreeLength = input.length === 3;
+    const isExistZero = input.some(x => x === '0');
+    if (isDuplicate || !isThreeLength || isExistZero) {
+      return false;
+    }
+    return true;
+  }
+
+  handleSubmit() {
+    this.submitBtn.addEventListener('click', () => {
+      const input = this.input.value.split('');
+      if (this.isCorrectInput(input)) {
+        this.showResult(this.play(this.randomNumbers, input));
+      } else {
+        window.alert('잘못된 입력값입니다.');
+      }
```

Comment:
```suggestion
      if (this.isCorrectInput(input)) {
        return this.showResult(this.play(this.randomNumbers, input));
      }
      
      window.alert('잘못된 입력값입니다.');
```

처럼 얼리 리턴을 해주는게 가독성에 더 좋을 것 같습니다~